stories/core/Table.stories.tsx

```diff
@@ -735,7 +738,9 @@ export const RowInViewport: Story<TableProps> = (args) => {
         <a
           className='iui-anchor'
           onClick={() =>
-            parent.document.getElementById('tabbutton-actions')?.click()
+            (parent.document.querySelector(
+              '[id^="tabbutton-actions"]',
```

Comment:
Why this was needed to change?

Reply:
the id 'tabbutton-actions' was changed to 'tabbutton-actions-{count}' so I changed the selector to use `^` (target prefix)